nimare/meta/cbma/base.py

```diff
@@ -151,6 +154,13 @@ def _fit(self, dataset):
         """
         self.dataset = dataset
         self.masker = self.masker or dataset.masker
+
+        if not isinstance(self.masker, NiftiMasker):
```

Comment:
Can you cover this in one of the tests?

Reply:
Do you know of any case in which this line is reached? If I provide a non-NiftiMasker to the estimator, it will get converted to a NiftiMasker by `get_masker(mask)`.

Reply:
`get_masker` should preserve the type of any nilearn masker (e.g., `NiftiMasker`, `NiftiLabelsMasker`, `NiftiMapsMasker`), so you _should_ be able to feed in a NiftiLabelsMasker and it would then raise this exception.

Reply:
Great. Thanks!